src/js/components/channels/twilio-channel-content.jsx

```diff
@@ -1,34 +1,138 @@
+                                         </button> : '';
+        const unlinkedComponent = <div>
+                                      <ReactTelephoneInput ref={ (c) => this._telInput = c }
+                                                           defaultCountry='ca'
+                                                           onChange={ this.handleInputChange }
+                                                           onValid={ this.onNumberValid }
+                                                           onInvalid={ this.onNumberInvalid }
+                                                           preferredCountries={ ['ca', 'us'] }
+                                                           onBlur={ this.handleInputBlur } />
+                                      { linkButton }
+                                  </div>;
+        const pendingComponent = <div className='twilio-linking pending-state'>
+                                     <i className='fa fa-phone'
+                                        style={ iconStyle }></i>
+                                     <span className='phone-number'>{ number } - Pending</span>
+                                     <a onClick={ this.onRetry }>Retry</a>
+                                 </div>;
+
+        const sendTextUrl = `sms://${phoneNumber}`;
+
+        const mobileLinkedComponent = <div>
+                                          <div className='twilio-linking unconfirmed-state'>
+                                              <i className='fa fa-phone'
+                                                 style={ iconStyle }></i>
+                                              <span className='phone-number'>{ number }</span>
+                                              <a onClick={ this.onChangeNumber }>Change my number</a>
+                                          </div>
+                                          <a href={ sendTextUrl }>
+                                              <button className='btn btn-sk-primary twilio-linking'
+                                                      onClick={ this.onStartTexting }> Start Texting </button>
+                                          </a>
+                                      </div>;
+        const desktopLinkedComponent = <div>
+                                           <div className='twilio-linking linked-state'>
+                                               <i className='fa fa-phone'
+                                                  style={ iconStyle }></i>
+                                               <span className='phone-number'>{ number }</span>
+                                               <a onClick={ this.onChangeNumber }>Change my number</a>
+                                           </div>
+                                           <a href={ sendTextUrl }>
+                                               <button className='btn btn-sk-primary twilio-linking'
```

Comment:
I'll guess we'll wait for me to merge the new settings, but we need to remember that these buttons will need to use the link color and contrast detection